### ISSUE TITLE
bug(ui): the quadlet name is not clickable

### ISSUE THREAD

**Issue by axel7083** [podman-desktop/extension-podman-quadlet#217]
## Description

When trying to open the Quadlet Details, we cannot click on the service name: this is an accessibility issue, as we always are able to click on the name in Podman Desktop
